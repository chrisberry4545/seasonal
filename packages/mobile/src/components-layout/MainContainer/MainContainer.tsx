import React from 'react';
import {
  styles
} from '../../styles';
import { ViewStyle, View } from 'react-native';
import { FC } from 'react';

const styleMainContainer: ViewStyle = {
  backgroundColor: styles.colors.backgroundColor,
  flex: 1,
  paddingLeft: 20,
  paddingRight: 20,
  paddingTop: 10
};

export const MainContainer: FC<{}> = ({
  children
}) => (
  <View style={ styleMainContainer }>
    { children }
  </View>
);
