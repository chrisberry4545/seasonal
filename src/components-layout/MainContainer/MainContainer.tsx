import React from 'react';
import {
  styles
} from '../../styles';
import { ViewStyle, View } from 'react-native';
import { SFC } from 'react';

const styleMainContainer: ViewStyle = {
  backgroundColor: styles.colors.backgroundColor,
  flex: 1,
  padding: 20
};

export const MainContainer: SFC<{}> = ({
  children
}) => (
  <View style={ styleMainContainer }>
    { children }
  </View>
);
