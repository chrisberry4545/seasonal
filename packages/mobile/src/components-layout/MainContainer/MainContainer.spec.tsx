import React from 'react';
import { MainContainer } from './MainContainer';
import { shallow, ShallowWrapper } from 'enzyme';

describe('<MainContainer />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <MainContainer>
        Child
      </MainContainer>
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());
});
