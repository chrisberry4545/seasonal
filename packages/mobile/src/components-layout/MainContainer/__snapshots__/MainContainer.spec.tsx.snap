// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContainer /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingTop": 10,
    }
  }
>
  Child
</View>
`;
